### ISSUE TITLE
Ui suggestion

### ISSUE THREAD

**Issue by bastiencyr** [borgbase/vorta#982]
Hello, 
I checked the mainWindow ui file because I wanted to optimize the space at the bottom (on my screen, the window is to big the first time). There is a status bar and and it seems that noboy use it so I have removed it. I have also moved the cancel and backup buttons to a toolBar. 
This is the result :
![image](https://user-images.githubusercontent.com/63359575/118247637-b12b3580-b4a3-11eb-9c90-9a51e88b43a6.png)

It's just a suggestion but I think a toolBar is a good idea. It is more compact and beautiful (with icons). 
My "Backup" button on toolBar starts a backup and it works.

I have others suggestions like adding the misc tab in a setting action in toolBar (all ui is made for one profile, one backup but misc tab is about all profiles and backups,  its global settings), keep misc tab for repository settings and add compression option in it, maybe move the profile menu elsewhere...

**Comment by m3nu** [comment]
Agree that the status bar could be removed or used for something.

Moving buttons to the menu might be a bit unexpected for users.

**Comment by bastiencyr** [comment]
I was thinking for example of Timeshift, it's clearer with a big button.  
Is it possible to integrate this project to propose and implement other ideas ?

**Comment by m3nu** [comment]
You mean like this screenshot? Pull requests that improve Vorta are always welcome. Just discuss your ideas before implementing. And dont copy code directly from other projects. Taking ideas and inspirations is ok though. 👌

![](https://res.cloudinary.com/practicaldev/image/fetch/s--jQ1drHaj--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/i/wif9rs60am8l5lg01fy3.png)

**Comment by bastiencyr** [comment]
Yes just like this. It's just inspiration. 
So yes, you have a toolbar with the most important option. 

I have other idea. I started to implement this. Look at this : 
So this is vorta without modification : 
![image](https://user-images.githubusercontent.com/63359575/118351249-2b6ac100-b55b-11eb-90db-541ddc0c70ae.png)

Actually, there is two modifications : the status bar has been removed and I added the toolbar. We already have talked about that. But notice that box are spaced. I think that the interface is is not sufficiently spaced. So look at this new suggestion : 

![image](https://user-images.githubusercontent.com/63359575/118351377-f4e17600-b55b-11eb-8822-1f55b2d67daa.png)

Maybe you see my new source of inspiration hh (settings of keepassXC). It was really not simple to do this. The icons are not the right icons.  It's the main modification but I have added many others minor modification.  Notice that on Linux, it seems to be more compact that on Mac without modifications. The tab organised vertically (for clock, wifi, tasks, terminal for example) makes these tabs difficult to see/ 

For example, I reduced the space at the bottom, so the text take only two spaces in heights instead of 3. I added spaces In misc, for wifi items,  clock, repository.
